test/docker-in-docker/docker_build.sh

```diff
@@ -9,8 +9,7 @@ source dev-container-features-test-lib
 check "docker-buildx" docker buildx version
 check "docker-build" docker build ./
 
-check "installs docker-compose v1 install" bash -c "type docker-compose"
-check "installs compose-switch" bash -c "[[ -f /usr/local/bin/compose-switch ]]"
```

Comment:
You would be able to revert these changes after you add back compose switch installation!